scripts/create-kubestellar-demo-env.sh

```diff
@@ -84,35 +117,78 @@ checking_cluster() {
 
     done
 }
+
+# Function to check if a command exists
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
 ##########################################
 
 echo -e "\nStarting environment clean up..."
 echo -e "Starting cluster clean up..."
 
-cluster_clean_up "kind delete cluster --name kubeflex" &
-cluster_clean_up "kind delete cluster --name cluster1" &
-cluster_clean_up "kind delete cluster --name cluster2" &
+# Check for k8s_platform and perform cleanup in the correct order
```

Comment:
What difference does the order make?

Reply:
if creating kind you need to delete k3s first. If creating k3s you need to create kind first

Reply:
What goes wrong if I use a different order?

Reply:
On delete. Nothing. On create it causes problems. The comment will be removed.

Reply:
And the logic updates. Don't need two loops

Reply:
fixed